src/server/handlers/launchpad.js

```diff
@@ -381,13 +382,24 @@ export const findSnaps = (req, res) => {
   const urlPrefix = getRepoUrlPrefix(owner);
   internalFindSnapsByPrefix(urlPrefix)
     .then((snaps) => {
-      return res.status(200).send({
-        status: 'success',
-        payload: {
-          code: 'snaps-found',
-          snaps: snaps
-        }
-      });
+      return Promise.all(snaps.map((snap) => {
+        return getSnapName(snap.git_repository_url, req.session.token)
+          .then((snapName) => {
+            return {
+              ...snap,
+              snap_info: { name: snapName }
```

Comment:
Whole `snap` object is "snap info", so while it doesn't matter much how we name it maybe some other name would be more appropriate? `yaml` or `snapcraft_yaml`? - we also should make sure we don't collide with snap API props names (not to override something by mistake).